DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Factories/GovernorFactory.cs

```diff
@@ -1,16 +1,17 @@
 using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Extensions;
 using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models.Gias;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models.Mstr;
 
 namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Factories;
 
 public interface IGovernorFactory
 {
-    Governor CreateFrom(GiasGovernance giasGovernance);
+    Governor CreateFrom(GiasGovernance giasGovernance, MstrTrustGovernance? mstrGovernance);
```

Comment:
This shouldn't be a nullable argument - there will always be a corresponding `MstrTrustGovernance`